### DISCUSSION TITLE
After upgrade to prisma 2.7.1, Error: ETXTBSY: text file is busy

### DISCUSSION THREAD

**Discussion by alan345** [prisma/prisma#3733] [Q&A] [answered]
After upgrade to prisma `2.7.1`, I got this error:

```
Error: 
Error: ETXTBSY: text file is busy, copyfile '/home/ubuntu/nacho/server/node_modules/@prisma/cli/query-engine-debian-openssl-1.1.x' -> '/home/ubuntu/nacho/server/node_modules/.prisma/client/query-engine-debian-openssl-1.1.x'
```

![image](https://user-images.githubusercontent.com/15246526/93908348-5abe7c00-fcb3-11ea-80c5-49022e1c84cb.png)

I rolled back to `2.7.0` and it works

**Comment by ryands17** [comment, marked as answer]
Hey @alan345 👋 
It would be great if you could create an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=). Also did you update both the Client and the CLI to 2.7.1 as I am not able to reproduce this on my system.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
